### PR TITLE
Add some examples of configuration options

### PR DESCRIPTION
**Pull request by sweeney**
Add a couple of examples into ravelinjs documentation, which automagically gets inserted into [developer.ravelin.com](https://developer.ravelin.com)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,6 +179,37 @@ a source of entropy. If insufficient events have been collected before
 encryption is attempted, an Error is thrown to prevent insecure transmission of
 cardholder data.
 
+## Examples
+
+### Device ID Prefix
```
Comment (adamroyjones) [inline README.md]: The heading here doesn't match the heading in the table of contents.

File: README.md [hunk 2]
```diff
@@ -176,6 +179,37 @@ a source of entropy. If insufficient events have been collected before
...
+
+RavelinJS prefixes the deviceId with `rjs-` by default. If you wish to use
+something else, for example if you are [upgrading](#upgrading) from a previous
+version and wish to maintain the opaque string format, simply specify your desired
+prefix or omit entirely:
+
+```
```
Comment (adamroyjones) [inline README.md]: We could add js to the end of the backticks to highlight the syntax. I'd use a GitHub suggestion here but their parser has a bug in it.

File: README.md [hunk 3]
```diff
@@ -176,6 +179,37 @@ a source of entropy. If insufficient events have been collected before
...
+var rav = new Ravelin({key: 'publishable_key_...', prefix: ''})
+```
+
+or
+
+```
```
Comment (adamroyjones) [inline README.md]: (and here)

File: README.md (hunk 2 above)
Comment (sweeney) [reply README.md]: TIL!

**Review by adamroyjones — APPROVED** [review]

**Pull request merged**